packages/data-store/src/types/digitalCredential/digitalCredential.ts

```diff
@@ -0,0 +1,46 @@
+export type NonPersistedDigitalCredential = Omit<DigitalCredential, 'id'>
+
+export type DigitalCredential = {
+  id: string
+  credentialType: CredentialType
```

Comment:
Suggest to rename this one to documentType, as we soon will also need the actual VC types in here for easy access, and then it become really confusing. Since a VP isn't a credential anyway and since documentFormat and uniformDocument are also already being used, suggest to stick with that

Reply:
renamed to `documentType`